web/cypress/integration/route/import_export_route.spec.js

```diff
@@ -124,14 +124,13 @@ context('import and export routes', () => {
 
   it('should delete the route', function () {
     cy.visit('/routes/list');
+    cy.get(this.domSelector.refresh).click();
+
     for (let i = 0; i < 2; i += 1) {
-      cy.contains(data[`route_name_${i}`])
-        .siblings()
-        .contains(componentLocaleUS['component.global.delete'])
-        .click();
-      cy.contains('button', componentLocaleUS['component.global.confirm']).click();
+      cy.contains('Delete').should('be.visible').click();
```

Comment:
There will be two delete buttons here, and you should locate the specific delete button IMO.

Reply:
I don't think there is a need to find a specific delete button. Contains will only find the first element that meets the conditions.
reference: https://docs.cypress.io/api/commands/contains.html#Single-Element

Reply:
I know the rule(only find the first element), but it's too tricky to write this way and not very friendly to new contributors IMO🤔

Reply:
Would better to add more information about this Button.

Reply:
All right.